code/index.html

```diff
@@ -1,37 +1,51 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="style.css" />
-    <title>Technigo Project Cluedo</title>
-  </head>
-  <body>
-    <section class="cards">
-      <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
-        <p id="killerName"></p>
-        <p id="killerAge"></p>
-        <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
-        <p id="killerDescription"></p>
-      </article>
-      <article class="weapon-deck" id="weaponCard">
-        <p>WEAPON</p>
-        <p id="weaponName"></p>
-        <p id="weaponWeight"></p>
-      </article>
-      <article class="room-deck" id="roomCard">
-        <p>ROOM</p>
-        <p id="roomName"></p>
-      </article>
-    </section>
-
-    <h1 id="mystery"></h1>
-
-    <button>REVEAL THE CRIME</button>
-
-    <script src="clue.js"></script>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" />
+  <title>Cluedo-Project</title>
+  <link href="https://fonts.googleapis.com/css?family=Luckiest+Guy|Roboto:300&display=swap" rel="stylesheet">
+
+</head>
+
+
+<body>
+  <h1 id="headingRefresh">Play Cluedo!</h1>
+  <h2>Click on each card and reveal the mystery</h2>
+  <section class="cards">
+    <article class="killer-deck" id="killerCard">
+      <p class="info-text">KILLER</p>
+      <p class="info-text" id="killerName"></p>
+      <p class="info-text" id="killerAge"></p>
+      <p class="info-text" id="killerOccupation"></p>
+      <img class="killer-image" id="killerImage" src="" />
+      <p class="info-text" id="killerDescription"></p>
+    </article>
+
+    <article class="weapon-deck" id="weaponCard">
+      <p class="info-text">WEAPON</p>
+      <p class="info-text" id="weaponName"></p>
+      <p class="info-text" id="weaponWeight"></p>
+    </article>
+
+    <article class="room-deck" id="roomCard">
+      <p class="info-text">ROOM</p>
+      <p class="info-text" id="roomName"></p>
```

Comment:
After seeing that you are adding `info-text` to all paragraphs in the `<section>` you got target those with `.cards p {}` instead